hydrate/cluster.py

```diff
@@ -64,3 +83,33 @@ def get_deployments_for_all_namespaces(self):
             deployment_list.append(d)
 
         return deployment_list
+
+
+def count_first_word(dict_list, key):
+    """Count the first word of each dict[key] in the list.
+
+    Args:
+        dict_list: List of dictionaries
+        key: Used to obtain the values from each dictionary
+
+    Returns:
+        dict(key:word, value:count) sorted by value desc. order
+
+    """
+    ret_count = dict()
+    for item in dict_list:
+        words = item[key].split("-")
+        ret_count[words[0]] = ret_count.get(words[0], 0) + 1
+    return ret_count
+
+def sort_dict_by_value(d):
+    """Sorts a dictionary by value.
+
+    Args:
```

Comment:
@andrewDoing an alternative to writing static documentation is to use [function annotations](https://www.python.org/dev/peps/pep-3107/) in Python. There are some benefits listed [here](https://www.python.org/dev/peps/pep-3107/#use-cases)  cc @edaena @mtarng